sql/driver/conn.go

```diff
@@ -103,16 +92,13 @@ func (c *conn) send(args Request) (*rows, error) {
 	// Set the session state even if the server returns an application error.
 	// The server is responsible for constructing the correct session state
 	// and sending it back.
-	if c.session != nil {
-		panic("connection has lingering session state")
-	}
 	c.session = resp.Session
 	// Translate into rows
-	r := &rows{}
+	resultRows := &rows{}
```

Comment:
yes, that's true. I believe I'm the first person to touch this since it was written, and the previous variable names made it really hard to make sense of the code, and even to look for the use of the identifiers. I think making that easier for future readers is worth the churn.